src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.ExitCommand;
+import duke.exception.DukeException;
+import duke.ui.TextUi;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+
+/**
+ * Drives the personal assistant using a UI, along with a parser to make sense of user inputs and executing commands on
```

Comment:
It would be better if this comment was split into two lines.

Reply:
Thanks! Will separate it for better readability. Such stylistic stuff was left to automated formatters so I honestly didn't consider looking out for it.